app/models/bulk_download.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require 'zip'
+
+##
+# This class is used to create a zip file
+# of all the files in a FileSet
+class BulkDownload
+  attr_accessor :zip_file_name, :files
+  ##
+  # @param [String] ID for an ActiveFedora::Base work with FileSets
+  # This zips up all the files in a FileSet object
+  def initialize(work_id)
+    work = ActiveFedora::Base.find(work_id)
+
+    @files = work.file_sets.map { |file_set| file_set.files }
+    @zip_file_name = Rails.root.join('tmp', Time.now.in_time_zone.to_a.join + '.zip')
+    Zip::File.open(zip_file_name, Zip::File::CREATE) do |zipfile|
+      @files.each do |file|
+        temp_file = Tempfile.new(file[0].file_name[0])
```

Comment:
Calling `file[0]` will only get the first file in the FileSet.  I don't think the first file is guaranteed to be the original file.  We might be serving up a random derivative.  I'm guessing that we should just include all the files in the FileSet, or else we'll have to add some logic to find the original file.

Reply:
This includes all the files from the file_set, it's a nested array so there are multiple `@files` that each have a `file[0]`